usefulUtilities/modelJSONCleanup/theSpot/theSpotNodeFilter.js

```diff
@@ -0,0 +1,194 @@
+// Consts
```

Comment:
this file looks very similar to the generic filter. can you please movidy the generic script so that it takes a JSON config that modifies the necessary variables so that we don't have two files that look/act very similar?

Reply:
Microsoft word.